README.md

```diff
@@ -222,8 +222,8 @@ For File storage we use the S3 buckets service provided by supabase. We do this
 
 ### Items
```

Comment:
```suggestion
### Assets
```

(for clarity?)

Reply:
You are correct. I re-adjusted it and deploying now again.